expo-zustand-styled-components/src/components/FilterDropdown/FilterDropdown.tsx

```diff
@@ -49,19 +49,16 @@ const FilterDropdown = ({
             <CloseIcon color={colors.gray600} width={20} height={20} />
           </TouchableOpacity>
         </DropdownOptionsHeading>
-        <FlatList
-          data={items}
-          renderItem={({ item }) => (
-            <DropdownOption onPress={() => selectOption(item)}>
-              {selected === item ? (
-                <CorrectIcon color={colors.gray600} />
-              ) : (
-                <View style={{ marginRight: 16 }} />
-              )}
-              <Text>{item}</Text>
-            </DropdownOption>
-          )}
-        />
+        {items.map((item) => (
```

Comment:
I am just curious (And maybe to learn also) but is there a reason why you decide not to use the `<FlatList />` component?

Reply:
I think I added a comment in one of the places I used map() instead of FlatList. FLatlist is mostly recommended when you have huge data and you want to lazy load them into the UI when the user scrolls. This isn't needed here as the dropdown can only have a limited number of data. Also, another issue with FlatList is, using it inside of a scroll view.